packages/safe-core-sdk-types/src/types.ts

```diff
@@ -59,6 +58,26 @@ export interface SafeTransaction {
   encodedSignatures(): string
 }
 
+export interface Transaction {
+  to: string
+  value: string
+  data: string
+  from?: string
+  gas?: number | string
+  gasLimit?: number | string
+  gasPrice?: number | string
+  maxFeePerGas?: number | string
+  maxPriorityFeePerGas?: number | string
+  nonce?: number
+}
+
+export interface MultiSendTransaction {
```

Comment:
This is `MetaTransactionData` at line 21

Reply:
done!